R/utils_render_rtf.R

```diff
@@ -1555,6 +1665,7 @@ create_footer_component_rtf <- function(data) {
 
   col_widths <-
     col_width_resolver_rtf(
+      page_orientation = dt_options_get_value(data = data, option = "page_orientation"),
```

Comment:
This and the same table_width calculation code in create_header_component_rtf could be refactored into a common function